ansible/library/ec2_group_facts.py

```diff
@@ -0,0 +1,251 @@
+#!/usr/bin/python
```

Comment:
I did not carefully review this file - I assume it's copied in from somewhere else?  If that's the case, can we add something to the commit message saying so (ideally with a link to where it came from)?